base.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM  ubuntu
```

Comment:
We should probably pin this to a specific tag so the CI environment stays constant across all runs. It's possible for the Ubuntu image to update and break the CI.

Reply:
It is picking latest, but it can be changed to preferred one. 20.04 LTS may be?

Reply:
> It is picking latest, but it can be changed to preferred one.

It's less about preference and more about stability of the CI pipeline. The `latest` tag points to `20.04`[0], but Canonical can always change that to a version with a breaking change (say change in package naming) and now we have to fix our CI pipeline, taking up valuable developer time. 

> 20.04 LTS may be?

An LTS is fine.

[0] https://hub.docker.com/_/ubuntu/

Reply:
changing it to 20.04 LTS